streaming_jupyter_integrations/examples/basic.ipynb

```diff
@@ -1,81 +0,0 @@
-{
```

Comment:
Examples removed?

Reply:
Those are duplicates, examples are in `${project_root}/examples` directory instead of `${project_root}/streaming_jupyter_integrations/examples`.
Not mentioning the fact that those in `streaming_jupyter_integrations/examples` are stale and plain wrong :d